app/admin/dashboard.rb

```diff
@@ -163,35 +287,24 @@
       end # tab
 
       tab :metrics do
+        # export .xlsx button
+        form action: export_metrics_path(), method: :get, style: 'text-align: right' do |f|
```

Comment:
for shame! `()` :smile: lol

Reply:
oops!! hahaha habits are hard to break -- my bad 😅